sdk/dotnet/Pulumi.Automation/PublicAPI.Unshipped.txt

```diff
@@ -312,6 +314,7 @@ override Pulumi.Automation.LocalWorkspace.SetConfigAsync(string stackName, Syste
 override Pulumi.Automation.LocalWorkspace.SetConfigValueAsync(string stackName, string key, Pulumi.Automation.ConfigValue value, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task
 override Pulumi.Automation.LocalWorkspace.WhoAmIAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<Pulumi.Automation.WhoAmIResult>
 override Pulumi.Automation.LocalWorkspace.WorkDir.get -> string
+static Pulumi.Automation.LocalWorkspace.CheckVersionIsValid(Semver.SemVersion minVersion, Semver.SemVersion currentVersion) -> void
```

Comment:
Not sure how to test this functionality without making this a public method.

Reply:
Since we aren't intending for this method to be accessible publicly, the "correct" way  would be to mark it `internal` and then to expose internal methods to the test library. Which is what the assembly attribute `InternalsVisibleTo(assemblyName)` does, which I think we're already doing - if not it's an easy add

Reply:
Aha, thank you! I did not know of `internal`, only `public` and `private`.